src/mono/wasm/build/WasmApp.targets

```diff
@@ -30,6 +30,17 @@
       - $(WasmAssemblySearchPaths)          - used for resolving assembly dependencies
       - $(WasmSkipMissingAssemblies)        - Don't fail on any missing dependencies
 
+      - $(WasmGenerateAppBundle)            - Controls whether an app bundle should be generated.
```

Comment:
Out of curiosity, for the use cases that don't build the wasmApp, do you know what the main application of this targets file would be?

Reply:
AOT, for example.

Reply:
Eg. blazor, that has it's own packaging stuff, but might want to AOT the assemblies.